.eslintrc.json

```diff
@@ -1,24 +1,23 @@
 {
-  "extends": [
-    "./node_modules/kyt/config/.eslintrc.base.json"
-  ],
+    "extends": [
+        "./node_modules/kyt/config/.eslintrc.base.json"
+    ],
 
-  "env": {
-  },
+    "env": {
+    },
 
-  "plugins": [
-  ],
+    "plugins": [
+    ],
 
-  "rules": {
-      "react/prefer-es6-class": 0,
-      "react/prefer-stateless-function": 0,
-      "object-shorthand": 1,
-      "comma-dangle": 0,
-      "indent": ["error", 2, { "SwitchCase": 2, "VariableDeclarator": 2, "outerIIFEBody": 2 }],
-      "react/jsx-indent-props": ["error", 2],
-      "react/jsx-indent": ["error", 2],
-      "object-shorthand": ["off", "always"],
-      "func-names": ["off", "always"]
-
-  }
+    "rules": {
+        "react/prefer-es6-class": 1,
```

Comment:
Well, kind of. We'll have to do an ES6 session at some point.

Btw, you can officially approve this PR by going to "Files Changed" tab. On the top right there is a "Review changes" button, and inside you can approve :)